ChattoAdditions/sources/Common/ScreenMetric.swift

```diff
@@ -158,6 +174,25 @@ extension UIScreen {
         return self.defaultKeyboardHeight(isPortrait: UIDevice.current.orientation.isPortrait)
     }
 
+    public func defaultKeyboardHeightForCurrentOrientation(
+        defaultOrientationAsPortrait: Bool = true
```

Comment:
Do you think the parameter purpose would be clearer as `fallbackUnknownOrientationAsPortrait`?